rclpy/rclpy/__init__.py

```diff
@@ -35,8 +35,8 @@ def init(args=None):
     return _rclpy.rclpy_init(args if args is not None else sys.argv)
 
 
-def create_node(node_name):
-    node_handle = _rclpy.rclpy_create_node(node_name)
+def create_node(node_name, namespace=None):
```

Comment:
913698ee392c73114d3a3409f94f162962b314ec

Note that the other functions here are not "keyword-only arguments".